xarray/backends/api.py

```diff
@@ -788,6 +789,33 @@ def open_dataarray(
     return data_array
 
 
+def open_datatree(
+    filename_or_obj: str | os.PathLike[Any] | BufferedIOBase | AbstractDataStore,
+    engine: T_Engine = None,
+    **kwargs,
+) -> DataTree:
+    """
+    Open and decode a dataset from a file or file-like object, creating one Tree node for each group in the file.
+    Parameters
+    ----------
+    filename_or_obj : str, Path, file-like, or DataStore
+        Strings and Path objects are interpreted as a path to a netCDF file or Zarr store.
+    engine : str, optional
+        Xarray backend engine to us. Valid options include `{"netcdf4", "h5netcdf", "zarr"}`.
+    kwargs :
+        Additional keyword arguments passed to :py:meth:`~xarray.open_dataset` for each group.
```

Comment:
This method doesn't exist yet. at that location.

Reply:
I think it doesn't exist at the top level because we wanted to migrate over the code before allowing a direct import.

``` python
from xarray import open_datatree
```

Until all of the code was merged, we said it would be imported from xarray.core.datatree.  From #8572 First comment `"EDIT: We decided it should return an xarray.DataTree object, or even xarray.core.datatree.DataTree object. So we can start by just copying the basic version in datatree/io.py right now which just calls open_dataset many times."`

So I had thought xarray.core.datatree.DataTree made more sense for now.

Reply:
I guess what I'm asking is: `open_dataset` definitely exists (and should be possible to link to), so I suppose you meant to write `xarray.open_datatree`?

Also, once that's fixed, does this cause the docs build to fail? If not I believe it would be fine to leave as-is.

Reply:
On reflection, I think this should still be open_dataset.  That is what is in the original docs. And this is describing open_datatree, where it's calling open_dataset with these kwargs each time.  So I think this should be reverted.  As to :py:meth: vs :py:func: I don't know and will look at the difference unless you tell me before I figure it out.